docs/IE_DG/Extensibility_DG/Extension.md

```diff
@@ -1,29 +1,29 @@
 # Extension Library {#openvino_docs_IE_DG_Extensibility_DG_Extension}
 
 Inference Engine provides an InferenceEngine::IExtension interface, which defines the interface for Inference Engine Extension libraries.
-All extension libraries should be inherited from this interface. The example below contains implementation of two operations: `Template`
+Inherit all extension libraries from this interface. The example below contains an implementation of two operations: `Template`
 used as an example in this document and `FFT` used as a more complex example from the [Custom Operations Guide](../../HOWTO/Custom_Layers_Guide.md).
 
-> **NOTE**: `FFT` operation is implemented using OpenCV library functions `cv::dft` and `cv::idft`.
+> **NOTE**: `FFT` operation is implemented using the OpenCV library functions `cv::dft` and `cv::idft`.
 
-Based on that, declaration of an extension class can look as follows:
+Based on that, the declaration of an extension class can look as follows:
 
 @snippet template_extension/extension.hpp extension:header
 
-The extension library should contain and export the method InferenceEngine::CreateExtension, which creates an `Extension` class:
+The extension library should contain and export the InferenceEngine::CreateExtension method, which creates an `Extension` class:
 
 @snippet template_extension/extension.cpp extension:CreateExtension
 
 Also, an `Extension` object should implement the following methods:
 
-* InferenceEngine::IExtension::Release deletes an extension object
+* InferenceEngine::IExtension::Release deletes an extension object.
 
-* InferenceEngine::IExtension::GetVersion returns information about version of the library
+* InferenceEngine::IExtension::GetVersion returns information about the version of the library.
 
 @snippet template_extension/extension.cpp extension:GetVersion
 
-Implement the  InferenceEngine::IExtension::getOpSets method if the extension contains custom layers. 
+Implement the InferenceEngine::IExtension::getOpSets method if the extension contains custom layers. 
 Read the [guide about custom operations](AddingNGraphOps.md) for more information.
```

Comment:
Should we use capital characters in the link names here and in lines 28 and 29?

Reply:
Changed to page names